src/components/ec-panel/ec-panel.vue

```diff
@@ -32,7 +32,7 @@
           aria-label="Close panel"
           class="ec-panel__header-action ec-panel__header-action--close"
           data-test="ec-panel__header-action--close"
-          @click.stop.prevent="closePanel"
+          @click.once.stop.prevent="closePanel"
```

Comment:
I think we need to address this problem differently. I see two issues with fixing it like this:
1. `closePanel` emits `close` event and it's up to the parent to decide if the panel should get closed or not. If parent will decide to not closing the panel, then it stays open and it will the close event will not be fired in the future.
2. Our code should be robust enough to handle multiple callback calls. It should not rely on the fact that the close event will be emitted 0..1 times. We should definitely check what is the problem in EBO's code and only if we cannot fix that, we will re-consider this approach.

I still don't understand what caused the error in EBO.

Reply:
I think like Milos, we should dig deeper into it, I'd prefer not to ship this fix.

Reply:
Yep, agree, this is more a workaround than a proper fix. I don't really understand the problem. Anyway I thought about sharing the PR to help understand the problem.